clients/src/main/java/org/apache/kafka/clients/consumer/internals/AsyncKafkaConsumer.java

```diff
@@ -849,9 +953,22 @@ public OptionalLong currentLag(TopicPartition topicPartition) {
         }
     }
 
+    public void setGroupMetadata(final ConsumerGroupMetadata groupMetadata) {
```

Comment:
This is just for testing, right? Can we remove the `public` qualifier?

Reply:
That was a temporary change that I missed to remove.